src/udev/udev-node.c

```diff
@@ -201,7 +203,8 @@ static int link_find_prioritized(sd_device *dev, bool add, const char *stackdir,
                         r = free_and_strdup(&target, devnode);
                         if (r < 0)
                                 return r;
-                }
+                } else
+                        continue;
```

Comment:
maybe debug log here? failures here are unexpected, no? hence maybe log about the error?

Reply:
> Shouldn't this be higher than `debug`?

Not sure, but the function is something like a library function now, and the similar errors are ignored. I'd like to keep it the debug level.